Makefile

```diff
@@ -29,8 +29,7 @@ test-grader-host:
 test-nocoverage: start-support
 	@mocha tests/index.js
 test-python:
-	@python3 question-servers/freeformPythonLib/prairielearn_test.py
-	@python3 elements/pl-order-blocks/dag_checker_test.py
+	@python3 -m pytest
```

Comment:
This doesn't matter in CI because the `graders` directory isn't included in the Docker image where tests are run, but it does impact me running `make test-python` locally.

```suggestion
# `pl_unit_test.py` has an unfortunate file name - it matches the pattern that
# pytest uses to discover tests, but it isn't actually a test file itself. We
# explicitly exclude it here.
	@python3 -m pytest --ignore graders/python/python_autograder/pl_unit_test.py
```

Reply:
@neneal can you please update the makefile to include the comment I suggested here? Documenting the rationale behind edge cases like this is really important.

Reply:
Added.